sky/clouds/azure.py

```diff
@@ -151,14 +173,8 @@ def region_zones_provision_loop(
         accelerators: Optional[Dict[str, int]] = None,
         use_spot: bool,
     ) -> Iterator[Tuple[clouds.Region, List[clouds.Zone]]]:
-        del accelerators  # unused
-
-        if instance_type is None:
-            # fallback to manually specified region/zones
-            regions = cls.regions()
-        else:
-            regions = service_catalog.get_region_zones_for_instance_type(
-                instance_type, use_spot, clouds='azure')
+        regions = cls.regions_with_offering(instance_type, accelerators,
+                                            use_spot, None, None)
```

Comment:
(styleguide) literals passed in as args are should use keywords to read easier

Reply:
Fixed.